automatic/lightalloy/update.ps1

```diff
@@ -0,0 +1,27 @@
+import-module au
+
+$releases = 'http://www.light-alloy.ru/download/'
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(?i)(^\s*packageName\s*=\s*)('.*')" = "`$1'$($Latest.PackageName)'"
```

Comment:
wrong regex match, should probably be `(?i)(^[$]packageName\s*=\s*)('.*')` instead.